core/modules/system/system.theme.inc

```diff
@@ -178,6 +180,9 @@ function theme_system_modules_fieldset($variables) {
     if ($required_by) {
       $requirements .= '<div class="admin-requirements">' . t('Required by:') . ' ' . implode(', ', $module['#required_by']) . '</div>';
     }
+    if ($modules_data[$key]->info['project'] !== 'backdrop') {
```

Comment:
Since we changed the variable name earlier, this also needs to be updated:

```suggestion
    if ($module_data[$key]->info['project'] !== 'backdrop') {
```